tensorboard/backend/event_processing/io_wrapper.py

```diff
@@ -31,6 +31,8 @@
 
 _ESCAPE_GLOB_CHARACTERS_REGEX = re.compile("([*?[])")
 
+directory_cache = {}
+use_bfs_and_build_cache_MT = True
```

Comment:
should this be a flag, or default false?

Reply:
This will be overriden by --concurrent_dir_discover in core_plugin.py.